Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM quay.io/wikiwatershed/taudem:5.3.8
+
+MAINTAINER Azavea <systems@azavea.com>
+
+ENV RWD_VERSION 1.2.0
+
+COPY requirements.txt /tmp/
+COPY src/api/requirements.txt /tmp/src/api/
+RUN pip install --no-cache-dir -r /tmp/requirements.txt && \
+    pip install --no-cache-dir -r /tmp/src/api/requirements.txt && \
+    rm /tmp/requirements.txt && \
+    rm /tmp/src/api/requirements.txt
+
+ENV PYTHONPATH :$PYTHONPATH
```

Comment:
This line previously existed to add `rwd_drb` and `rwd_nhd` modules to the global Python path so they could be imported from the API. Now that you copy these modules to the same directory as the API this is no longer necessary.